Assets/HoloToolkit/Sharing/Tests/Scripts/ImportExportAnchorManager.cs

```diff
@@ -883,11 +876,12 @@ private void AnchorLoadComplete()
         /// </summary>
         private void Export()
         {
+            WorldAnchor anchor = this.GetComponent<WorldAnchor>();
```

Comment:
In accordance with the suggested change above, the changes in this function should be able to be reverted.